src/domain/signupGroup/__tests__/permissions.test.ts

```diff
@@ -71,6 +72,40 @@ describe('getSignupGroupActionWarning function', () => {
     }
   );
 
+  it('should return correct warning when trying to delete signup group with signup with payment cancellation', () => {
+    expect(
+      getSignupGroupActionWarning({
+        authenticated: true,
+        signupGroup: fakeSignupGroup({
+          signups: [
+            fakeSignup({ paymentCancellation: fakePaymentCancellation() }),
```

Comment:
Maybe use one more fake signup here without a cancellation to better prevent possible future modifications where the group's delete action condition is changed to something that is wrong (`some` -> `every`, etc). Same can be done to the refund test.

Reply:
I added an other signup with payment cancellation/refund null to prevent future modifications of the function.